views/pages/experimental/apply/form.njk

```diff
@@ -95,21 +95,29 @@
                                 data-conditional-fields='{{ field.conditionalFor | dump | safe }}'
                             {% endif %}
                             {% if field.isConditionalOn and fieldErrors | length < 1 %}data-conditional-field="{{ field.name }}" hidden{% endif %}>
-                            <div class="form-field__body">
-                                {% if field.type === 'text' %}
-                                    {{ inputText(field) }}
-                                {% endif %}
-                                {% if field.type === 'textarea' %}
-                                    {{ inputTextarea(field) }}
-                                {% endif %}
-                                {% if field.type === 'radio' %}
-                                    {{ inputRadio(field) }}
+                            <div class="form-field__main">
+                                <div class="form-field__body">
+                                    {% if field.type === 'text' %}
```

Comment:
If we find this pattern coming up a lot maybe we need a macro to do this switching for us? But fine for now.